lib/pdksync.rb

```diff
@@ -4,57 +4,29 @@
 require 'fileutils'
 require 'rake'
 require 'pdk'
-require 'pdksync/constants'
+require 'pdksync/configuration'
 require 'pdksync/gitplatformclient'
 require 'json'
 require 'yaml'
 require 'colorize'
 require 'bundler'
 require 'octokit'
+require 'pdk/util/template_uri'
```

Comment:
@logicminds Hi Corey , Just wondering if we need this line.
I commented the line and pdksync worked .

Otherwise it was throwing the following error
rake aborted!
LoadError: cannot load such file -- pdk/util/template_uri

Reply:
hmm.  What version of PDK do you have?